### ISSUE TITLE
Missing snap icon

### ISSUE THREAD

**Issue by jkfran** [canonical/docker-snap#47]
It would be nice to include the docker logo to the snap

https://snapcraft.io/docker

**Comment by tianon** [comment]
Unfortunately, the Docker logo is a trademark of Docker, Inc. and so it's not acceptable for us to use. :disappointed:

**Comment by jkfran** [comment]
@tianon I suspected this was the issue :)

Would it be possible to ask for permission? Did we try?

**Comment by tianon** [comment]
Yes, at one point Docker Inc was even maintaining the snap directly, but no longer.
